### DISCUSSION TITLE
simple question about history

### DISCUSSION THREAD

**Discussion by jck411** [danny-avila/LibreChat#3245] [Q&A] [answered]
does LibreChat retrieve history from previous conversations into new conversations? ...I am asking because I THOUGHT I had this working on one of my early installs but now its not, my mongo is showing all the collections and storing messages, mielsearch is also searching correctly. But when I start a new chat it doesn't pull from this. So is this an intrinsic property of LibreChat or do we code ourselves the middleware to fetch and inject historical data?...was I just hallucinating like AI ? lol or am I installing this wrong

**Comment by fuegovic** [comment, marked as answer]
LibreChat has never offered this feature... 
Every conversation is a fresh start

**Reply by jck411** [reply]
Thank you for confirming!
